homeassistant/components/dynalite/const.py

```diff
@@ -16,16 +16,43 @@
 CONF_AUTO_DISCOVER = "autodiscover"
 CONF_BRIDGES = "bridges"
 CONF_CHANNEL = "channel"
+CONF_CHANNEL_COVER = "channelcover"
 CONF_CHANNEL_TYPE = "type"
+CONF_CLOSE_PRESET = "close"
 CONF_DEFAULT = "default"
+CONF_DEVICE_CLASS = "class"
+CONF_DURATION = "duration"
 CONF_FADE = "fade"
 CONF_HOST = "host"
 CONF_NAME = "name"
 CONF_NO_DEFAULT = "nodefault"
-CONF_POLLTIMER = "polltimer"
+CONF_OPEN_PRESET = "open"
+CONF_POLL_TIMER = "polltimer"
 CONF_PORT = "port"
 CONF_PRESET = "preset"
+CONF_ROOM = "room"
+CONF_ROOM_OFF = "room_off"
+CONF_ROOM_ON = "room_on"
+CONF_STOP_PRESET = "stop"
+CONF_TEMPLATE = "template"
+CONF_TILT_TIME = "tilt"
+CONF_TIME_COVER = "timecover"
```

Comment:
Please separate words with underscore.
```suggestion
CONF_TIME_COVER = "time_cover"
```

Reply:
Just noticed that there are a few CONF options there that are already two words without an underscore ("autodiscover", "polltimer", and "nodefault"). I can change them to two words with underscore, but this would create a breaking change. should i do it?
In any case, changed also channelcover to channel_cover

Reply:
Let's don't change the existing ones, unless we make another breaking config change in the future.

Reply:
ok. thanks